app/controllers/merchants_controller.rb

```diff
@@ -0,0 +1,55 @@
+require 'pry'
+class MerchantsController < ApplicationController
+  def index
+    @merchants = Merchant.all
+  end
+
+  def show
+    @merchant = Merchant.find(params[:id])
+  end
+
+  def new
+  end
+
+  def create
+    Merchant.create(merchant_params)
+    redirect_to '/merchants'
+  end
+
+  def edit
+    @merchant = Merchant.find(params[:id])
+  end
+
+  def update
+    @merchant = Merchant.find(params[:id])
+    @merchant.update({
+      name: params[:name],
+      address: params[:address],
+      city: params[:city],
+      state: params[:state],
+      zip: params[:zip]
+      })
+    @merchant.save
+    redirect_to "/merchants/#{@merchant.id}"
+  end
```

Comment:
Same thing here as in the items controller... you can use the merchant_params method that you have below to clean this up a bit.